File: 1week/dajung/python/1213_짝수홀수개수.py
def solution(num_list):
    answer = []
    even = []
    odd = []
    for i in num_list:
        #짝수일 때
        if i%2 == 0:
            even.append(i);
        #홀수일 때
        else:
            odd.append(i);
    answer.append(len(even))
    answer.append(len(odd))    
    return answer

File: 1week/dajung/javaScript/1213_짝수홀수개수.js
function solution(num_list) {
    var answer = [];
    let even = [];
    let odd = [];
    for (let i=0; i<num_list.length; i++){
        if (num_list[i]%2 === 0){//짝수일 때
            even.push(num_list[i]);
        } else {//홀수일 때
            odd.push(num_list[i]);
        }
    }
    answer.push(even.length);
    answer.push(odd.length);
    return answer;
}